CONTRIBUTING.md

```diff
@@ -1,55 +1,43 @@
 ## Contribution Guide Overview
-This guide provides steps on how to make contributions and become a part of the ongoing development of `Socket Runtime`.  
+This guide provides steps on how to make contributions, report bugs, and become a part of the ongoing development of `Socket Runtime`.  
+If you are just started learning about Socket Runtime, you can start from [here](https://socketsupply.co/guides/).  
 
-### Requirements
+## Setup and build the framework
```

Comment:
```suggestion
## Requirements
```
Changing the header order is fine, but I think "Requirements" is more idiomatic.